README.md

```diff
@@ -685,6 +685,8 @@ to be `false` and all symbol names will be omitted.
 
 - `if_return` (default: `true`) -- optimizations for if/return and if/continue
 
+- `imports` (default: `true`) -- drop unreferenced imports through `unused`
```

Comment:
The wording seemed like the import itself was being dropped.  Perhaps:
```
drop unreferenced import symbols when used with `unused`
```

Reply:
Thanks &minus; updated in https://github.com/mishoo/UglifyJS/pull/4646/commits/4600fa7f4f1ecf2a0674a0673fdcf8a3b344822f